src/dashboards/components/AutoRefreshOverlay.tsx

```diff
@@ -0,0 +1,178 @@
+// Libraries
+import React, {FC, useContext} from 'react'
+
+// Components
+import {
+  Overlay,
+  Grid,
+  Button,
+  ComponentColor,
+  SelectDropdown,
+  SlideToggle,
+  InputLabel,
+  ComponentStatus,
+} from '@influxdata/clockface'
+import {OverlayContext} from 'src/overlays/components/OverlayController'
+import AutoRefreshDropdown from 'src/shared/components/dropdown_auto_refresh/AutoRefreshDropdown'
+import TimeRangeDropdown from 'src/shared/components/DeleteDataForm/TimeRangeDropdown'
+
+// Types
+import {CustomTimeRange, AutoRefreshStatus} from 'src/types'
+
+// Context
+import AutoRefreshContextProvider, {
+  AutoRefreshContext,
+} from 'src/dashboards/components/RefreshContext'
+
+import './AutoRefresh.scss'
+
+// This array creates an array of [0...24] for the hours selection
```

Comment:
I think it will be okay if we give the dropdown a height max and hide the rest w/scroll bar. 

Another option is to provide options that are multiple of 2 or 3 or some number rather than all 1~24. There's probably diminishing returns on that granular of a control on inactivity timeout.  

I think I like the idea of giving the dropdown a height max and hiding the rest w/scrollbar. What do you guys think?

Reply:
@daviddkkim the current implementation has a height and a scroll bar to hide the rest, so ill be keeping this the way it is since it sounds like you are on board with that